.github/workflows/workflow.yml

```diff
@@ -92,7 +92,9 @@ jobs:
       - name: install papermill
         if: ${{ env.RUN_NB_TESTS == '1' }}
         shell: bash -l {0}
-        run: pip install papermill[all]
+        run: |
+          conda install ipykernel
+          pip install papermill[all]
```

Comment:
https://anaconda.org/conda-forge/papermill
cant the pip install be replaced by conda forge install? 
```suggestion
```